approach.tex

```diff
@@ -242,8 +242,9 @@ \subsubsection{Procedural and Technical Limitations}
 \begin{itemize}
 
 
-  \item{The \product{} system must be verified before the complete LSST system can be completed. Verification is therefore carried out using precursor datasets~\footnote{e.g. from Hyper Suprime-Cam; \secref{LDM-503-02}}, simulated data, and --- where available --- with engineering and pre-release data from the as yet incomplete LSST system.}
-
+  \item{The \product{} system must be verified before the complete LSST system can be completed. Verification is therefore carried out using precursor datasets~\footnote{e.g. from Hyper Suprime-Cam; \secref{LDM-503-02}}, simulated data, and --- where available --- with engineering and pre-release data from the as yet incomplete LSST system~\footnote{e.g. from AuxTel, ComCam and LSSTCam commissioning activities;  \secref{LDM-503-08}, \secref{LDM-503-12a}}. Full verification of the \product{} system will depend critically on the availability of this data; if critical data is not available, it may not be possible to successfully complete verification of all the high-level requirements defined in \citeds{LSE-61} according to the specified prioritization, or at all before completion of the DM Construction Project. In such cases, requirements will be verified as best they can with the available data. The Test Plan and Report documents (\secref{sect:testdocs}) will make clear all deviations from the requirements due to the available data as well as any exercises that should be carried out once the necessary data becomes available to fully verify the requirement. 
```

Comment:
Minor formatting nit: it'd be easier to read the source if this was split across multiple lines.

Other than that: consider avoiding the repeated use of “critical” by saying “if appropriate data is not available”. I think that also avoids us having to specify up-front what that data is. (If we refer to “critical data”, the obvious question is “please identify up-front the data that is critical”; if we write “appropriate”, its easier to come back later and say “oh, no appropriate data was available”).